installation/sdxc_formatting.md

```diff
@@ -16,6 +16,8 @@ Run the Raspberry Pi Imager application, then from the `Choose OS option`, selec
 
 The standard formatting tools built into these operating systems are able to create FAT32 partitions; they might also be labelled as FAT or MS-DOS. Simply delete the existing exFAT partition and create and format a new FAT32 primary partition, before proceeding with the rest of the [NOOBS instructions](noobs.md). On a Mac, this means using the command-line `diskutil` program.
 
+NOTE: For Mac, use `Master Boot Record` for the scheme.
```

Comment:
Probably best to make this part of the preceding paragraph. Something like "On a Mac, this means using the command-line `diskutil` program: choose the `Master Boot Record` scheme."

Reply:
I agree, think it would scan better if it were in the preceding paragraph.